### DISCUSSION TITLE
Add organizational accounts field to include all repositories from accounts list

### DISCUSSION THREAD

**Discussion by andry81** [lowlighter/metrics#1195] [Ideas]
This is a continue of that: https://github.com/lowlighter/metrics/discussions/1146

I have organizational accounts to just group my repositories and have several to group repositories been committed mostly by scripts or bots. The metrics does include the storage has occupied only by repositories from the main account.

If try to to add `affiliations=''`, then it just adds all repositories, including repositories with mostly bot commits.

May be add a field to add other accounts including organizational additionally to field with skip repositories to calculate all repositories from list of accounts?

**Comment by lowlighter** [comment]
Could #1056 maybe suits your needs ?

What was discussed in it was the ability to pass multiples login and group stats together

**Reply by andry81** [reply]
> Could #1056 maybe suits your needs ?

I suppose those accounts for all the metrics. But I suggest to add organizations to calculate specific metric - repositories occupied size. This means that is not need to know which accounts included into organization to calculate the repositories occupied size. It could just use the list of organizations (or wildcard expression like "user-*" to select all organization accounts beginning by "user-" includes "user" as owner just for a start) and wildcard expression for repositories to select for calculation. Nothing more.

You can then add multiple lines into the report, like:

```
user:     XX MB
user-aaa: YY MB
user-bbb: ZZ MB
```

And so on for each organizational account beside the main account: `user`. Or accumulate it altogether:

<pre>
XX<sub>user</sub> MB + NNN<sub>orgs</sub> MB used
</pre>

**Reply by lowlighter** [reply]
So basically you'd like to see a detailed view of the storage for each organization if I'm understanding correctly ?

**Reply by andry81** [reply]
> So basically you'd like to see a detailed view of the storage for each organization if I'm understanding correctly ?

Yes, if it is possible. If not then, just at least the altogether.
